frontend/util/datetime.js

```diff
@@ -1,7 +1,26 @@
-import moment from 'moment';
+import { format, distanceInWords, distanceInWordsStrict } from 'date-fns';
+
 
 const NO_TIME = '-';
 
+/**
+ * Helper to get the given date or `now` in UTC.
+ * @param {String | Date} dateString  date to convert to UTC. Defaults to now.
+ * @return {Date}                     date in UTC
+ */
+function getUTCDate(dateString = Date.now()) {
+  const date = new Date(dateString);
+
+  return new Date(
```

Comment:
I wonder if there is a formatting helper so we don't have to manually call all these functions (event if it is encapsulated in this function 😄 )

Reply:
There is not one that I found. This was a recommended solution from an issue comment thread.